### PR TITLE
Volumetric support for RelateObjects (resolves #2624)

### LINKED ISSUE #2624: RelateObjects should support volumes.

### PR DESCRIPTION
**Pull request by mcquin**
https://github.com/CellProfiler/CellProfiler/pull/2700#issuecomment-295257739

### CODE REVIEW

File: cellprofiler/module.py [hunk 1]
```diff
@@ -938,7 +938,8 @@ def run(self, workspace):
         y = cellprofiler.image.Image(
             dimensions=dimensions,
             image=y_data,
-            parent_image=x
+            parent_image=x,
+            convert=False
```
Comment (mcquin) [inline cellprofiler/module.py]: Resolves #2687. `convert=True` caused an error in modules downstream of Erosion due to datatype.

**Comment by braymp** [comment]
Does this fix #2442?

**Comment by mcquin** [comment]
@braymp It does now! :)

**Comment by mcquin** [comment]
screen shot 2017-04-19 at 8 47 40 am

File: cellprofiler/module.py (hunk 1 above)
Comment (0x00b1) [inline cellprofiler/module.py]: 👍

File: cellprofiler/module.py [hunk 2]
```diff
@@ -972,24 +973,17 @@ def add_measurements(self, workspace, object_name=None):
...
-            centers = scipy.ndimage.center_of_mass(numpy.ones_like(labels), labels=labels, index=unique_labels)
-
-            centers = numpy.array(centers)
+            if objects.volumetric:
+                center_z, center_x, center_y = centers.transpose()
+            else:
+                center_z = [0] * len(centers)
 
-            center_z, center_x, center_y = centers.transpose()
+                center_x, center_y = centers.transpose()
```
Comment (0x00b1) [inline cellprofiler/module.py]: Should this be outside of the `else` branch?

Comment (mcquin) [inline cellprofiler/module.py]: Nope!

Comment (0x00b1) [inline cellprofiler/module.py]: Thanks :)

**Pull request merged**